### PR TITLE
Provisioning of 1.16.2

### PR DESCRIPTION
**Pull request by dhiller**
Signed-off-by: Daniel Hiller <daniel.hiller.1972@gmail.com>

### CODE REVIEW

File: cluster-provision/k8s/scripts/provision.sh [hunk 1]
```diff
@@ -38,14 +40,11 @@ gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
...
 EOF
 yum install -y \
-  docker-common-1.13.1-75.git8633870.el7.centos.x86_64 \
-  origin-docker-excluder-3.11.0-1.el7.git.0.62803d0.noarch \
-  python-docker-py-1.10.6-4.el7.noarch \
-  docker-client-1.13.1-75.git8633870.el7.centos.x86_64 \
-  cockpit-docker-176-2.el7.centos.x86_64 \
-  docker-1.13.1-75.git8633870.el7.centos.x86_64 \
-  python-docker-pycreds-1.10.6-4.el7.noarch
+  docker-1.13.1 \
```
Comment (phoracek) [inline cluster-provision/k8s/scripts/provision.sh]: how is this change related to the patch?

File: cluster-provision/README.md [hunk 2]
```diff
@@ -25,6 +25,8 @@
 * `kubevirtci/k8s-1.11.0:`: `sha256:696ba7860fc635628e36713a2181ef72568d825f816911cf857b2555ea80a98a`
 * `kubevirtci/k8s-1.13.3:`: `sha256:afbdd9b4208e5ce2ec327f302c336cea3ed3c22488603eab63b92c3bfd36d6cd`
 * `kubevirtci/k8s-1.14.6`: `sha256:ec29c07c94fce22f37a448cb85ca1fb9215d1854f52573316752d19a1c88bcb3`
+* `kubevirtci/k8s-1.15.1`: `sha256:14d7b1806f24e527167d2913deafd910ea46e69b830bf0b094dde35ba961b159`
```
Comment (phoracek) [inline cluster-provision/README.md]: why do you update 1.15 here?

Comment (dhiller) [reply cluster-provision/README.md]: I added it because the sha was missing.

File: cluster-provision/k8s/scripts/provision.sh (hunk 1 above)
Comment (dhiller) [reply cluster-provision/k8s/scripts/provision.sh]: I had to update the install as the packages requested previously weren't available any more. We agreed upon just installing whatever is available instead of using the exact packages.

Comment (rmohr) [reply cluster-provision/k8s/scripts/provision.sh]: Yep, and the background: We were pinning them back then because of bugs.

**Comment by rmohr** [comment]
/approve
/hold

Feel free to unhold when you feel confident about the results in kubevirt/kubevirt.
